src/aat/resources/features/F-108/GetToken_Autotest2.td.json

```diff
@@ -0,0 +1,69 @@
+{
+  "_guid_": "Private_Case_Creation_Autotest2_Data_Token_Creation",
+  "productName": "CCD Data Store",
+  "operationName": "Start event creation as Case worker",
+  "method": "GET",
+  "uri": "/caseworkers/{uid}/jurisdictions/{jid}/case-types/{ctid}/event-triggers/{etid}/token",
+
+  "specs": [
+    "a case that has just been created as in Private_Case_Creation_Data",
+    "a case that has just been created as in [Private_Case_Creation_Data]",
+    "to create a full case",
+    "to create a token for case creation"
+  ],
+
+  "users": {
+    "invokingUser": {
+      "username": "auto.test12.cnp+private@gmail.com",
+      "password": "Pa55word11"
+    }
+  },
+
+  "request": {
+    "headers":{
+      "Authorization": "[[DEFAULT_AUTO_VALUE]]",
+      "ServiceAuthorization": "[[DEFAULT_AUTO_VALUE]]",
+      "Content-Type": "application/json"
+    },
```

Comment:
`"__extends__": "CommonRequest"`
this will already bring these standard headers. please reuse them for better maintainability.